flag_engine/identities/models.py

```diff
@@ -19,6 +20,23 @@ class TraitModel:
     trait_value: typing.Any
 
 
+@contextmanager
+def override_identity_traits(
```

Comment:
Although I don't think it will make much sense to put this in utils/context.. that will kinda make utils dumping ground of  everything. I think I would rather create a submodule called `context_manager` in the identity module itself and if we have to create another context_manager for any other module that we can make it top level module, what do you think?

Reply:
Yep, makes sense.